documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1422,6 +1422,35 @@ This field is required to enable Oracle RAC support.
 Based on the used hash function referential integrity is kept while data is pseudonymized. Supported hash functions are described in the {link-java7-standard-names}[MessageDigest section] of the Java Cryptography Architecture Standard Algorithm Name Documentation.
 The hash is automatically shortened to the length of the column.
 
+|[[oracle-property-log-mining-batch-size-min]]<<oracle-property-log-mining-batch-size-min, `log.mining.batch.size.min`>>
+|1000
+|The minimum SCN interval size that this connector will try to read from redo/archive logs. Active batch size will be also increased/decreased by this amount for tuning connector throughput when needed.
+
+|[[oracle-property-log-mining-batch-size-max]]<<oracle-property-log-mining-batch-size-max, `log.mining.batch.size.max`>>
+|100000
+|The maximum SCN interval size that this connector will use when reading from redo/archive logs.
+
+|[[oracle-property-log-mining-batch-size-default]]<<oracle-property-log-mining-batch-size-default, `log.mining.batch.size.default`>>
+|20000
+|The starting SCN interval size that the connector will use for reading data from redo/archive logs.
+
+|[[oracle-property-log-mining-sleep-time-min]]<<oracle-property-log-mining-sleep-time-min, `log.mining.sleep.time.min`>>
```

Comment:
The option name should end with ".ms", as it's the convention for similar duration-kind properties.